extensions/8to16/CTDB.js

```
@@ -0,0 +1,96 @@
+        ]
+      };
+    }
+
+    async setKey(args) {
+      const key = encodeURIComponent(args.KEY);
+
+      await fetch(`https://ctdbapi.funstrangeegg.workers.dev/api/${key}`, {
+        method: "POST",
+        headers: { "Content-Type": "application/json" },
+        body: JSON.stringify(args.VALUE)
+      });
```

ESLint

[error] Use Scratch.fetch() instead of fetch()

(extension/use-scratch-fetch)